Description: Use system Python
Forwarded: not-needed
Origin: https://src.fedoraproject.org/rpms/onnxruntime/blob/f40/f/system-python.patch

--- a/cmake/CMakeLists.txt
+++ b/cmake/CMakeLists.txt
@@ -671,16 +671,16 @@
 if (onnxruntime_BUILD_SHARED_LIB OR onnxruntime_ENABLE_PYTHON)
   if (onnxruntime_ENABLE_PYTHON)
     if (onnxruntime_REQUIRE_PYTHON_EMBED_LIB)
-      find_package(Python 3.10 COMPONENTS Interpreter Development NumPy)
+      find_package(Python ${PYTHON_VERSION} COMPONENTS Interpreter Development NumPy)
     else()
-      find_package(Python 3.10 COMPONENTS Interpreter Development.Module NumPy)
+      find_package(Python ${PYTHON_VERSION} COMPONENTS Interpreter Development.Module NumPy)
     endif()
     message("Numpy version: ${Python_NumPy_VERSION}")
     if(Python_NumPy_VERSION VERSION_LESS "2.0.0")
       message(WARNING "The build binary will not be compatible with NumPy 2.0 because the NumPy installed on this machine is too low.")
     endif()
   else()
-    find_package(Python 3.10 COMPONENTS Interpreter)
+    find_package(Python ${PYTHON_VERSION} COMPONENTS Interpreter)
   endif()
 endif()
 
